prep-css-box-model/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  p {
+    margin: 20px;
+  }
+
+  border-width: 3px;
+  border-color: darkgray;
+  p.dotted { border-style: double; }
+
+  div {
+    padding: 10px;
+  }
```

Comment:
Hi Derek, you can remove the selector portion of these rule sets. no `p` or `div` or extra curly braces needed, just the `border-` settings so that your buttons look like the example.

![image](https://user-images.githubusercontent.com/7432943/128272755-533f7d93-579d-4d29-9de1-162cb064dda0.png)